WPFSKillTree/SkillTreeFiles/Compute.cs

```diff
@@ -2310,7 +2310,7 @@ public static void Initialize(SkillTree skillTree, ItemAttributes itemAttrs)
 
             CoreAttributes();
 
-            Implicit = new AttributeSet(skillTree.ImplicitAttributes(Global));
+            Implicit = new AttributeSet(SkillTree.ImplicitAttributes(Global, skillTree.Level));
```

Comment:
some clarification
var st = new SkillTree()...;
st._level; //SHOULD BE private field, I will change it
st.Level;//public property to acess st._level
SkillTree.ImplicitAttributes(...);//static method to compute Health, mana gains etc. from int/dex/str and level

The actual change i made was: 
public ... st.ImplicitAttributes(attribs) method on SkillTree instance, that used only 1 nonstatic field - "sk.Level".
Is changed it to public static ImplicitAttributes(attrib, level) method to which you now have to pass the actual level.

The implicit attribute gains are same for all character classes and skilltrees, so i don't see reason to place the method that does not use anything (==pasive nodes), except level, from actual character as instance method.